buildkite/pipelines/bacalhau-python.yaml

```diff
@@ -1,16 +1,12 @@
 
 
 steps:
-  - label: ":swagger: Generate Swagger"
-    command: "./scripts/generate_swagger.sh"
-
-  - wait: ~
 
   - label: ":python: Build Python API Client"
-    command: "make build-python-apiclient"
+    command: "./buildkite/scripts/build_python_client.sh"
 
   - label: ":python: Build Python SDK"
-    command: "make build-python-sdk"
+    command: "./buildkite/scripts/build_python_sdk.sh"
```

Comment:
what is the difference between these two?

Reply:
Python client is the client generated from swagger, whereas the sdk is the wrapper over our client. We publish both of those.